frontend/src/static/js/v_summary_charts.js

```diff
@@ -167,6 +167,24 @@ window.vSummaryCharts = {
       return fileTypes;
     },
 
+    getFileTypeContribution(ele) {
```

Comment:
Adding these 3 methods to `v_summary_charts` even though the methods exist in `v_summary`.

This is because both vue components are utilising these methods. And vue doesn't allow the child component to _call parent's method with a return value_.

Not sure if this is the best way of doing it.

Reply:
If we refactor out these three functions to `util.js`, and then call them in `v_summary` and `v_summary_charts` will it work then?